packages/parser/src/index.ts

```diff
@@ -1,7 +1,7 @@
 export { parse, parseForESLint, ParserOptions } from './parser';
 export {
   ParserServices,
-  clearCaches,
+  clearWatchCaches,
```

Comment:
It's probably best to continue exporting this as `clearCaches` and mark it as `@deprecated`.
otherwise technically this is a breaking change because I know there are some other plugins using this API in their tests
```suggestion
  /** @deprecated - use clearWatchCaches instead */
  clearCaches,
  clearWatchCaches,
```

Reply:
Ah right ok... I can also remove it entirely, was just a clarity point - I didn't want use confusing ourselves thinking that function affected the AOT program cache as well.

Just let me know if you think your suggestion here, or just reverting the change is preferable overall